vcf.c

```diff
@@ -6230,6 +6230,11 @@ static int64_t get_rlen(const bcf_hdr_t *h, bcf1_t *v)
         size = bcf_dec_size(f, &f, &type);
         f += size << bcf_type_shift[type];
     }
+
+    // Can skip SVLEN lookup if there are no <DEL> symbolic alleles
+    if (!use_svlen)
+        svlenid = -1;
+
```

Comment:
svlenid is initialized unnecessarily above if use_svlen==0, a table lookup can be avoided if placed here in an `else` statement

Reply:
True, although I note that we could skip searching for ALT symbolic alleles if there's no `SVLEN` in the header.  That might be quicker, for files that don't use `SVLEN`.